src/nim_status_client.nim

```diff
@@ -85,6 +85,28 @@ proc ensureDirectories*(dataDir, tmpDir, logDir: string) =
   createDir(tmpDir)
   createDir(logDir)
 
+proc logHandlerCallback(messageType: cint, message: cstring, category: cstring, file: cstring, function: cstring, line: cint) {.cdecl, exportc.} =
+  logScope:
+    topics = "qt"
+    text = $message
+    file = $file & ":" & $line
```

Comment:
If we are to have two `file` arguments, can we then rename this one `qmlFile`?

Reply:
I'll make it `qtFile`, as it's not only QML

Reply:
or even better, `source_file`